packages/k8s-triliovault-operator/k8s-triliovault-operator.patch

```diff
@@ -0,0 +1,32 @@
+diff -x '*.tgz' -x '*.lock' -uNr packages/k8s-triliovault-operator/charts-original/Chart.yaml packages/k8s-triliovault-operator/charts/Chart.yaml
+--- packages/k8s-triliovault-operator/charts-original/Chart.yaml
++++ packages/k8s-triliovault-operator/charts/Chart.yaml
+@@ -1,6 +1,7 @@
+ apiVersion: v1
+ appVersion: v2.0.2
+-description: K8s-TrilioVault-Operator is an operator designed to manage the K8s-TrilioVault Application Lifecycle.
++description: K8s-TrilioVault-Operator is an operator designed to manage the K8s-TrilioVault
++  Application Lifecycle.
```

Comment:
Not sure if this was intentional. If it was accidental, please remove it to avoid cluttering the patch, as it should only contain Rancher specific changes like the annotations below.

Reply:
This was accidental. @PennyScissors Do we need .patch file even when we are uploading the partner chart for the very first time? I thought, we need patch file only when we update the helm chart to new release or new product version.
Correct me my above understanding is wrong, if not, can we remove the .patch file from this PR?

Reply:
The patch itself is not required, but in the case of a partner chart you need it so that you can add the required annotations. The idea is the patch should only contain all Rancher specific changes that you need, so most partner charts only have the annotations in their patch.

Reply:
Thank you @PennyScissors . I have updated the files accordingly. Please check and let me know.